string name
string bond_id
uint32 connection_id
